src/containers/Home.js

```diff
@@ -34,10 +31,19 @@ class Home extends Component {
     }
 
     render() {
+        const { data, errorMessage }  = this.props;
+        console.log(data);
         return (
             <div>
                 <h1>REDDIT NEWS</h1>
                 <Header handleFetch={this.handleFetch} />
+
+                {/*Tell User to select a category, or display the news*/}
+                {data === undefined && !errorMessage 
```

Comment:
check instead if data is falsey? `!data` ? It could also be `null` or empty list `[]`